charts/kyverno/Chart.yaml

```diff
@@ -51,4 +51,4 @@ dependencies:
   - name: openreports
     version: "0.1.0"
     repository: "https://openreports.github.io/reports-api"
-    condition: openreports.enabled
+    condition: openreports.enabled,openreports.installCrds
```

Comment:
_Condition - The condition field holds one or more YAML paths (delimited by commas). If this path exists in the top parent's values and resolves to a boolean value, the chart will be enabled or disabled based on that boolean value. Only the first valid path found in the list is evaluated and if no paths exist then the condition has no effect._

Multiple conditions are always "or" linked. So only one condition needs to be true. In this case your expected behavior will not work as expected

Reply:
@fjogeleit
Thanks for pointing that out. I’ve made the changes and updated the description with the new behavior.